app/views/categories/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="header">
+  <div class="back">
+    <span class='back-arrow back-link cursor-li' id="menu"> &#9776;</span>
+  </div>
+  <div class="middle-title">
+    <h1>Categories</h1>
+  </div>
+  <div class="next">
+    &nbsp;
+  </div>
+</div>
+<div id="menuContainerDiv" class="menu-container">
+  <div class="link-deta">
+    <center>
+    <% if user_signed_in? %>
+      <span class="logout">
+        &#10226;
+        <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: 'logout-btn' %>
+      </span>
+    <% end %>
+    </center>
+  </div>
+</div>
+<div class="categories">
+  <center>
+    <p style="color: green"><%= notice %></p>
+  </center>
+  <br />
+  <% @categories.each do |category| %>
+    <%= render category %>
+  <% end %>
```

Comment:
- [ ] Kindly fix the category card elements placement and add the correct widths and margin values to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/228708898-79b0d8d3-aa83-42a3-b79c-3d2ab8540caa.png) | ![image](https://user-images.githubusercontent.com/68796232/228708803-199739c9-671b-409f-a790-9dbd07becd45.png)  |